SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LegacyLoadBalancingStrategy.cpp

```diff
@@ -227,6 +323,21 @@ void FLegacyLoadBalancing::Init(ISpatialOSWorker& Connection, FLoadBalancingShar
 		OutServerWorkerData.Add(ServerWorkerCustomAssignment.Get());
 	}
 #endif //! UE_BUILD_SHIPPING
+
+	OutLoadBalancingData.Add(AlwaysRelevantStorage.Get());
+	OutLoadBalancingData.Add(ServerAlwaysRelevantStorage.Get());
+
+#ifndef BENCH_INTEREST_PERF
```

Comment:
Would be good to wrap this into an actual functional test at some point? If possible?